spec/fixtures/files/dummy_repo/repodata/abf421e45af5cd686f050bab3d2a98e0a60d1b5ca3b07c86cb948fc1abfa675e-primary.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metadata xmlns="http://linux.duke.edu/metadata/common" xmlns:suse="http://novell.com/package/metadata/suse/common" xmlns:rpm="http://linux.duke.edu/metadata/rpm" packages="4">
+<package type="rpm">
+  <name>apples</name>
+  <arch>x86_64</arch>
+  <version epoch="0" ver="0.1" rel="0"/>
+  <checksum type="sha256" pkgid="YES">5c4e3fa1624bd23251eecdda9c7fcefad045995a9eaed527d06dd8510cfe2851</checksum>
+  <summary>Apples dummy RPM</summary>
+  <description>Dummy apples RPM</description>
+  <packager></packager>
+  <url></url>
+  <time file="1500453272" build="1500453253"/>
+  <size package="1934" installed="14" archive="268"/>
+<location href="apples-0.1-0.x86_64.rpm"/>
+  <format>
+    <rpm:license>Public</rpm:license>
+    <rpm:vendor>obs://build.suse.de/Devel:SCC:rmt-dummy</rpm:vendor>
```

Comment:
Do we want/need to have this info in a public repo?

Reply:
Yeah, sure -- it's just a fixture for specs. That repo with fixture files was built on OBS, but we don't care where it was build for anything RMT-related.